lib/sneakers_toolbox.rb

```diff
@@ -1,8 +1,39 @@
 require "sneakers_toolbox/version"
 require 'sneakers_toolbox/lost_db_connection_handler'
 require 'sneakers_toolbox/ticktock_worker'
+require 'dry-configurable'
 
 module SneakersToolbox
+  extend Dry::Configurable
+  setting :ticktock do
+    # Allows to define callbacks for various exception classes for all Ticktock
+    # workers in one place
+    #
+    # To be assigned a Hash where key is subclass of StandardError and value is
+    # callable (its #call method will be called)
+    #
+    # Example use:
+    #
+    # SneakersToolbox.config.ticktock.error_callbacks = { MyError => Proc.new { notify_my_service } }
+    setting(:error_callbacks, {}) do |callbacks|
+      if callbacks.keys.all? { |k| k < StandardError } && callbacks.values.all? { |v| v.respond_to?(:call) }
```

Comment:
```ruby
k < StandardError
```
What's this comparison doing?

Reply:
https://ruby-doc.org/core-2.2.0/Module.html#method-i-3C Detects if it's a subclass of the right side operand

Reply:
#TIL

Reply:
Now that you pointed my attention to it, I think it should use `<=` instead so users can also add callbacks for `StandardError`